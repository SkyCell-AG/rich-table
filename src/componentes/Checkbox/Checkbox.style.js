import {
    makeStyles,
} from '@mui/styles'

const useStyles = makeStyles(() => {
    return {
        root: {
            margin: 0,
        },
    }
})

export default useStyles
