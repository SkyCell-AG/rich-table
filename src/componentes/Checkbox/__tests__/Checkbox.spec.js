import React from 'react'
import {
    shallow,
} from 'enzyme'

import Checkbox from '../Checkbox'

describe('Checkbox', () => {
    it('should match snapshot', () => {
        const wrapper = shallow(
            <Checkbox
                title="title"
                onChange={jest.fn()}
            />,
        )

        expect(wrapper).toMatchSnapshot()
    })
})
