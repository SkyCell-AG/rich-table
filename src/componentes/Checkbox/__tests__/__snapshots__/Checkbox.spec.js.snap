// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox should match snapshot 1`] = `
<div
  className=""
>
  <ForwardRef(FormControlLabel)
    classes={
      Object {
        "root": "makeStyles-root-1",
      }
    }
    control={
      <ForwardRef(Checkbox)
        checked={false}
        color="primary"
        disabled={false}
        indeterminate={false}
        name=""
        onChange={[Function]}
        value={false}
      />
    }
    label="title"
  />
</div>
`;
